test/cypress/utils/request_to_go_live_utils.js

```diff
@@ -60,11 +70,30 @@ const setupStubs = (serviceRole) => {
   cy.task('setupStubs', simpleStub(serviceRole))
 }
 
+const stubUserSuccessResponse = (serviceRolesBefore, serviceRolesAfter) =>
+  [{
+    name: 'getUserSuccessRepeatFirstResponseNTimes',
+    opts: [{
+      external_id: variables.userExternalId,
+      service_roles: [serviceRolesBefore],
```

Comment:
This is just a single role you're passing in as a parameter, so the variable names should probably be `serviceRoleBefore` and `serviceRoleAfter`